README.md

```diff
@@ -1 +1,189 @@
-# React_Sneakers_Store
\ No newline at end of file
+<div align="center">
+<h1 align="center">Games Gallery</h1>
+</div>
+
+
+
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+    - [🚀 Live Demo](#live-demo)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Run tests](#run-tests)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Games Gallery] `React` `Redux` `API` `ES6`<a name="about-project"></a>
+
+
+**Games Gallery**  is a Single Page Application (SPA) designed for users to explore games by category and view details of their chosen games. We provide an exciting and up-to-date experience for gaming enthusiasts.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <ul>
+    <li>Front-end: React, React Redux, React Router,JavaScript.</li>
+    <li>API Integration: Free-to-Play Games Database for real-time data.</li>
+    <li>Version Control: GitHub with Gitflow.</li>
+    <li>Testing: React Testing Library, Jest.</li>
+    <li>Build Tool: Create React App.</li>
+  </ul>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Home**
+  
+The Home section serves as the main landing page, offering users the ability to explore and filter games by category. It provides an overview of available game categories and allows users to click on a specific category to view details about games within that category. This section is designed for users to easily discover and select games based on their interests.
+
+- **Details**
+ 
+The Details section provides in-depth information about a selected game from the chosen category. Users can access details such as the game's title, description, platform availability, developer, release date, and more. It serves as a dedicated space for users to get comprehensive insights into the games they are interested in and make informed decisions about which games to explore further.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://react-game-gallery-5dd33.netlify.app/)
+
```

Comment:
- [ ] Great job working on this milestone. As per project requirements, you are supposed to add the link your video presentation which is missing at this time. Kindly record a video presentation and add the link.
![Screen Shot 2023-09-06 at 5 20 33 AM](https://github.com/JavierAybar/React_Games_Gallery/assets/62849872/f72984a9-7350-4584-b428-5db568ea7451)